sample/perf/index.html

```diff
@@ -56,10 +56,12 @@
     await populate(rep, opts);
     bench.reset();
     await rep.query(async (tx) => {
-      let i = 0;
+      var count = 0;
```

Comment:
why var?

Reply:
Because I sux. Looks like you or I already fixed this.